### ISSUE TITLE
Focus management after submitting a valid contact code [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#390]
Upon submitting a valid contact code on the **Share Your Close Contact Codes** screen, new instructional text appears below the text fields, but focus is moved to the first **Close Contact Codes** input field, which announces the value, in this case "3, edit box, disabled".
 
**Remediation Guidance**
We suggest moving focus to the new instructional text, as it is not immediately apparent to TalkBack users. Upon hearing the “3, edit box, disabled” announced after form submission, users may incorrectly assume they have been taken to a new screen. Or at the very least, have to swipe through each (disabled) Close Contact Codes input fields to reach the instructions.

**Comment by JohnAGill** [comment]
I addressed this issue on ios which should have worked on android too

**Comment by jonnyjames** [comment]
@JohnAGill I won't be able to confirm the fix until I am supplied with another Close Contact Code.

**Comment by floridemai** [comment]
@jonnyjames Who provided you codes before? I can also do that for you, just ping on the thread please

**Comment by AlanSl** [comment]
@helloantoine is reporting he also sees the issue on Android, but not on IOS. 

I'll pick it up next and make sure it's also fixed on Android.

**Comment by helloantoine** [comment]
This works on iOS but issue remains on Android.

**Comment by AlanSl** [comment]
Should be fixed in the next build (45)

**Comment by jonnyjames** [comment]
@AlanSl I still have not received new **Close Contact Codes** to retest this functionality. Could someone on your team issue me a few, via email? jjames@tpgi.com Thanks!

**Comment by AlanSl** [comment]
@jonnyjames we'll send you a few later. 

Build 45 also isn't out yet, I've started the build process so hopefully it should be ready in an hour or so.

**Comment by AlanSl** [comment]
@jonnyjames Build 45 is out now, I'll send you some upload codes shortly

**Comment by jonnyjames** [comment]
This issue is now resolved. Closing this issue.
